markets/bfp-market/test/integration/modules/PerpAccountModule.mergeAccounts.test.ts

```diff
@@ -510,34 +550,34 @@ describe('PerpAccountModule mergeAccounts', () => {
   it('should merge two accounts', async () => {
     const { BfpMarketProxy, MergeAccountSettlementHookMock } = systems();
     const { fromTrader, toTrader } = await createAccountsToMerge();
-
-    const collateral = collaterals()[1];
+    // Use nonUSD collateral to make sure we still have some debt. And use a generator to make sure we have two different collaterals for the fromAccount.
```

Comment:
Formatting here please. Too long.

I would also split this test into two distinct tests, `should merge accounts`, `should merge accounts when fromAccount has multiple collaterals` for explicitness. You can probably break this out into more tests with a test for when there is no debt on fromId and add a test for when there's only sUSD as this only generates a non sUSD backed positions.

Reply:
Yea happy to break it out. The only way we wont have debt is if we use sUSD collateral though, as otherwise the order fee will be added to the debt